pruneArcBuild.js

```diff
@@ -0,0 +1,27 @@
+const glob = require("glob");
+var fs = require("fs");
+
+const files = glob.sync("./node_modules/@daostack/arc/build/contracts/*.json", {
+  nodir: true
+});
+
+console.log(`Starts pruning Arc JSON files`);
+
+files.filter(file => {
+  const contract = require(`${file}`);
+  console.log(`Pruning ${contract.contractName}`);
+  var cleanVersion = '{\n "contractName": "';
+  cleanVersion += contract.contractName;
```

Comment:
Seems like a weird and error prone way to do it. I would:
```
const {contractName, abi, bytecode, deployedBytecode} = JSON.parse(fileContent)
fs.writeFileSync(pathToFile, JSON.stringify({contractName, abi, bytecode, deployedBytecode}, undefined, 2), 'utf-8')
```

Reply:
Fixed